src/md5-browser.js

```diff
@@ -19,10 +19,14 @@
  * See http://pajhome.org.uk/crypt/md5 for more info.
  */
 function md5(bytes) {
-  if (typeof bytes == 'string') {
-    var msg = unescape(encodeURIComponent(bytes)); // UTF8 escape
-    bytes = new Array(msg.length);
-    for (var i = 0; i < msg.length; i++) bytes[i] = msg.charCodeAt(i);
+  if (typeof bytes === 'string') {
+    const msg = unescape(encodeURIComponent(bytes)); // UTF8 escape
+
+    bytes = new Uint8Array(msg.length);
```

Comment:
Why a Uint8Array instead of a plain Array?

Reply:
In general, typed arrays should work faster, since the JS engine will immediately understand what data it works with.

https://bugs.chromium.org/p/v8/issues/detail?id=2190

You can see the tests at this link ^

Reply:
https://v8.dev/blog/elements-kinds

Reply:
Nice.  When I first added v3/v5 support, I wanted to insure support for older, legacy platforms that didn't have typed array support.  That's no longer a concern.